content/models/social_force_model/index.md

```diff
@@ -0,0 +1,118 @@
+
+In the figure above, the black dot represents an agent. Its radius is represented by the gray circle around it. The desired direction is towards the target, which is indicated by an X.
+
+## Agent force
+The agents exert a repulsive force on each other, which increases exponentially with their proximity. When they collide the pushing forces are increased additionally. When colliding, a frictional force also occurs, which acts orthogonally to the repulsive force in the direction of the speed difference. 
+
+$$ \overrightarrow{F\_i^{\mathrm{ped}}} = \sum_{j} \overrightarrow{f_{ij}}$$
+
+$$\overrightarrow{f_{ij}} = [A_i exp[(r_{ij} - d_{ij})/B_i] + kg(r_{ij} - d_{ij})] \overrightarrow{n_{ij}} + \kappa g(r_{ij} - d_{ij}) \Delta v\_{ji}^t \overrightarrow{t_{ij}}$$
+
+$$\phantom{......}\fbox{\phantom{...................} pushing \phantom{...................}} \fbox{\phantom{......} sliding \phantom{......}}$$
+
+* $j$ is another agent
+* $r_{ij} = r_i + r_j$ is the combined radius of $i$ and $j$
+* $d_{ij}$ is the distance between $i$ an $j$ them. 
+* $A_i$, $B_i$, $k$ and $\kappa$ are constants 
+  -  $A_i$ agent scale
+  -  $B_i$ force distance
+  -  $k$ body force
+  -  $\kappa$ friction
+* $g$ is a function with $g(x) =  \begin{cases}
+                                      x \text{ if } x < 0 \\
+                                      \text{ else } 0
+                                  \end{cases}$
+* $ \overrightarrow{n_{ij}} = (n\_{ij}^1, n\_{ij}^2) = (\overrightarrow{pos\_i} - \overrightarrow{pos\_j})/d\_{ij} $ is the normalised vector from $j$ to $i$ 
+* $ \overrightarrow{t_{ij}} = (- n\_{ij}^2, n\_{ij}^1)$ is the tangent of $\overrightarrow{n_{ij}}$ which is perpendicular to it, rotated counterclockwise
+* $\Delta v\_{ji}^t = (\overrightarrow{v\_j} - \overrightarrow{v\_i}) \cdot \overrightarrow{t_{ij}}$ is the tangential velocity difference of $i$ and $j$
+
+{{< figure src="Pushing_Agents.png" caption="Direction of repulsive forces acting on an agent" width="500">}}
+
+The repulsive force of the agents acts from the originating agent towards the agent on which the forces act. When their distance greater than their combined radius no other forces apply
+
+{{< figure src="Sliding_Agent_Forces.png" caption="Direction of the pushing and frictional forces acting on colliding agents">}}
+
+If the distance between two agents is smaller than their combined radius, the agents collide and a frictional force also occurs. This frictional force acts orthogonally to the repulsive force in the direction of the velocity difference of the two agents.
+
+## Obstacle force
+Obstacles exert a force on the agent similar to a static agent.
+The repulsive force is increases exponentially with decreasing distance. When the Obstacle collides with the agent the pushing forces are increased additionally. When colliding, a frictional force also occurs, which acts orthogonally to the repulsive force in the direction of the speed of the agent.
```

Comment:
Obstacle -> obstacle

Actually, the agent collides with the obstacle not the other way around